ASF-STM.user.js

```diff
@@ -1164,6 +1165,7 @@
                 errors = 0;
                 debugPrint("processing page " + page);
                 updateMessage("Processing badges page " + page);
+                updateProgress(page, maxPages, 0);
```

Comment:
you are using maxPages before it's assigned a value. Once again, you are lucky that JS is shit and don't mind.